conans/test/functional/graph/private_deps_test.py

```diff
@@ -309,62 +309,3 @@ def test_consumer_private(self):
                          self.client.out)
         self.assertNotIn("Hello1/0.1@lasote/stable: Generating the package",
                          self.client.out)
-
```

Comment:
Is this something related to the two profiles mechanism?

Reply:
Yes, this is a bug of the "private" requires, with the 2 profiles they are treated like a "build_requires", and it will not add itself to the ``self.deps_cpp_info``. And all of this will change in Conan 2.0 graph model, so probably I thought it was not worth trying to fix it, but maybe I can try to fix it for Conan 1.X.